### PR TITLE
Adapt cabin sorting endpoints

### PR DESCRIPTION
**Pull request by NelsonDong41**
Hot fix for previous merge:

- Functionality for remove all functions in judging algorithm daos were removed during merge. Reimplemented here

### CODE REVIEW

**Review by alyssamui — APPROVED** [review]

**Review by mike10911 — APPROVED** [review]
Looks great. Fix merge conflicts

File: backend/cabin-sorting/tsconfig.json [hunk 1]
```diff
@@ -1,109 +1,46 @@
 {
   "compilerOptions": {
```
Comment (surbhi-gulati) [inline backend/cabin-sorting/tsconfig.json:2]: is there a reason to include ALL of these settings?

File: backend/cabin-sorting/hackerSortingAlgo.ts [hunk 2]
```diff
@@ -110,9 +120,9 @@ function hackerSortingAlgo () {
   } else {
     matchAnswers()
     printMembers()
-    writeDataToFile()
+    writeDataToFile()    
   }
 }
 
 // let's get sorting!!
-hackerSortingAlgo()
```
Comment (surbhi-gulati) [inline backend/cabin-sorting/hackerSortingAlgo.ts:118]: don't call this directly in the file, call within your main sorting process. similar callout to https://github.com/HackBeanpot/internal-tools/pull/137/files#diff-2b2de8166d35c5f271eae616c700b6841e5920da821789a78a1ac62963e760f3

**Review by surbhi-gulati — CHANGES_REQUESTED** [review]
you can remove your name from the PR title if that's alright! just two small comments, looking good. thanks for moving fast on it!

File: backend/cabin-sorting/tsconfig.json (hunk 1 above)
Comment (NelsonDong41) [reply backend/cabin-sorting/tsconfig.json:2]: I believe I transferred the options over from judging algo but it does work without all of the included options so i reverted it to what I had

File: backend/cabin-sorting/hackerSortingAlgo.ts (hunk 2 above)
Comment (NelsonDong41) [reply backend/cabin-sorting/hackerSortingAlgo.ts:118]: The main sorting process is done in the hackerSortingAlgo.ts file, there is no main file that calls the script like in judging algo's index.ts for the parser function. For this one I would have to leave this function here I believe

**Pull request merged**